docassemble/assemblylinewizard/test_map_names.py

```diff
@@ -1,46 +1,48 @@
 import unittest
 
-from .interview_generator import map_names
+from interview_generator import map_names
 from docassemble.base.util import log
 
 __all__ = ['TestMapNames']
 
 class TestMapNames(unittest.TestCase):
- 
+
     def setUp(self):
         pass
 
-    def mapped_tests(self):
-      # Look in the console for a prettier version of the messages
-      errored = []
-      passed = []
-      for scenario_input in scenarios:
-        log('~~~~~~~~~~', 'console')
-
-        # Add our result to the collection
-        try:
-          desired_output = scenarios[ scenario_input ]
-          result = self.assertEqual( desired_output, map_names( scenario_input ) )
-          log( result, 'console' )
-          passed.append(scenario_input)
-
-        # The error should show us what specifically didn't match up
-        except AssertionError as error:
-          log( error, 'console' )
-          errored.append({ "test": scenario_input, "result": error })
-
-      results = {"errored": errored, "passed": passed}
-      log(results, 'console')
-      return results
- 
-if __name__ == '__main__':
+    def test_mapped_scenarios(self):
```

Comment:
Will `generator-test.yml` still function? If not, we should probably delete it in this same PR.

Reply:
Even better - I can make this same test work for both use cases. Does that work?